dtos/event.go

```diff
@@ -82,8 +82,8 @@ func (e *Event) AddBinaryReading(resourceName string, binaryValue []byte, mediaT
 }
 
 // AddObjectReading adds a object reading to the Event
-func (e *Event) AddObjectReading(resourceName string, objectValue interface{}) {
-	e.Readings = append(e.Readings, NewObjectReading(e.ProfileName, e.DeviceName, resourceName, objectValue))
+func (e *Event) AddObjectReading(resourceName string, valueType string, objectValue interface{}) {
```

Comment:
don't break the existing public API.  
create another function call `AddObjectArrayReading`

Reply:
Added new func.